modal/_functions.py

```diff
@@ -648,10 +650,17 @@ def from_local(
             for method_name, partial_function in partial_functions.items():
                 function_type = get_function_type(partial_function.is_generator)
                 function_name = f"{info.user_cls.__name__}.{method_name}"
+
+                if not partial_function._is_web_endpoint():
+                    method_schema = get_callable_schema(partial_function.raw_f, ignore_first_argument=True)
```

Comment:
Minor but might make some code easier to follow if we had a positive assertion for ... whatever kind of think you are when you're a method with a callable schema that isn't a web endpoint.